fsspec/spec.py

```diff
@@ -944,6 +944,10 @@ def open(self, path, mode="rb", block_size=None, cache_options=None, **kwargs):
         """
         Return a file-like object from the filesystem
 
+        The file-like object returned ignores an eventual PrefixFileSystem:
```

Comment:
I wouldn't put this here, the vast majority of people calling this won't be using Prefix. I would put it into the docstring of prefix's open()